src/Emanote/Model/Link/Rel.hs

```diff
@@ -80,6 +80,7 @@ unresolvedRelsTo r =
 parseUnresolvedRelTarget :: [(Text, Text)] -> Text -> Maybe UnresolvedRelTarget
 parseUnresolvedRelTarget attrs url = do
   guard $ not $ "://" `T.isInfixOf` url
+  guard $ not $ "mailto:" `T.isInfixOf` url
```

Comment:
I wonder if we should replace both of this with a simple `":" `T.isInfixOf` url` ... because, we do want to ignore other protocols like ftp://, magnet:, etc.

Consequently, this would mean the note filenames cannot have `:` in them. I'll look to see if this is really necessary or can be avoided.

Reply:
Addressed in https://github.com/srid/emanote/commit/35ccd46ac6c83d3ead775eb9e3595def9db69c46